### DISCUSSION TITLE
Not working statically the useStranslations

### DISCUSSION THREAD

**Discussion by vitalijalbu** [amannn/next-intl#1080] [Show and tell]
<img width="878" alt="Screenshot 2024-05-21 alle 17 02 47" src="https://github.com/amannn/next-intl/assets/40239902/b877700a-4541-444b-a841-f861ce50d88f">

Basically I followed each step from:

https://next-intl-docs.vercel.app/docs/getting-started/app-router/without-i18n-routing

**Comment by vitalijalbu** [comment]
Everything is ok but when i type:
`const t = useTranslations();` app crashes...


```
import PageContainer from '@/components/common/page-container';
import { useTranslations } from 'next-intl';

const Page = async () => {
const t = useTranslations();


  return (
    <PageContainer>
     ciao
    </PageContainer>
  );
};

export default Page;
```

**Reply by amannn** [reply]
You can't use hooks like `useTranslations` in async Server Comonents (see https://github.com/amannn/next-intl/issues?q=is%3Aissue+suspended+thenable+is%3Aclosed). You can use [`getTranslations`](https://next-intl-docs.vercel.app/docs/environments/server-client-components#async-components) there instead.
